backend/core/src/primary-adapters/rest/video/video-controller.ts

```diff
@@ -305,55 +322,4 @@ export class VideoController extends BaseHttpController {
 
     return res;
   }
-
```

Comment:
Did you delete comment reaction endpoint?

Reply:
fixed it, that I did not solve the conflicts correctly